tests/std/tests/P0896R4_ranges_ref_view/test.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <span>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+// clang-format off
+template<class Range>
+concept can_empty = requires (Range&& range) { ranges::empty(range); };
+// clang-format on
+
+struct instantiator {
+
+    template <class Read>
+    static constexpr void call() {
+        using ranges::ref_view, ranges::iterator_t, ranges::sentinel_t, ranges::equal, ranges::begin, ranges::end;
+        int input[3] = {0, 1, 2};
+
+        { // traits
+            STATIC_ASSERT(ranges::input_range<Read> || ranges::output_range<Read, const int&>);
+            STATIC_ASSERT(ranges::enable_borrowed_range<ref_view<Read>>);
+        }
+
+        { // constructors
+            STATIC_ASSERT(!constructible_from<ref_view<Read>, Read>);
+
+            [[maybe_unused]] ref_view<Read> default_constructed{};
+            STATIC_ASSERT(is_nothrow_default_constructible_v<ref_view<Read>>);
+
+            Read wrapped_input{input};
+            [[maybe_unused]] ref_view<Read> same_range{wrapped_input};
+        }
+
+        { // access
+            Read wrapped_input{input};
+            ref_view<Read> test_view{wrapped_input};
+            auto& base_range = test_view.base();
+            STATIC_ASSERT(same_as<decltype(base_range), Read&>);
+            assert(addressof(base_range) == addressof(wrapped_input));
+
+            STATIC_ASSERT(noexcept(test_view.base()));
+        }
+
+        { // iterators
+            Read wrapped_input{input};
+            ref_view<Read> test_view{wrapped_input};
+            const same_as<iterator_t<Read>> auto first = test_view.begin();

```

Comment:
We should verify that all of these accessors (all the members are accessors) are callable on a `const ref_view`. Coverage for mutable as well is ok, but I don't think mandatory - we're unlikely to break mutable calls with a member that's callable on `const` objects.